common/changes/@microsoft/rush/master_2019-06-06-23-07.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "comment": "Fix git process in rush version",
```

Comment:
Could we make this changelog give a better indication of what is impacted?  Maybe something like this:

`Improve "rush version" to fetch before checkout, which avoids an error in cases where the branch wasn't fetched`

Reply:
Updated.